Src/FluentAssertions.Json/JTokenAssertions.cs

```diff
@@ -90,9 +90,9 @@ public JTokenAssertions(JToken subject)
 
             var message = $"JSON document {difference?.ToString().Escape(true)}.{Environment.NewLine}" +
                           $"Actual document{Environment.NewLine}" +
-                          $"{Format(Subject, true).Escape(true)}{Environment.NewLine}" +
+                          $"{Format(Subject, true).Replace("{", "{{").Replace("}", "}}")}{Environment.NewLine}" +
```

Comment:
Could you add tests to verify the fixes?
This helps showing what is currently broken and that regressions are not introduced.

Reply:
Just upgrading makes existing tests fail: `When_objects_are_not_equivalent_it_should_throw`
Because it will print \r\n instead of going to the next line.

Reply:
Ah... I assume this is due to https://github.com/fluentassertions/fluentassertions/pull/987, which changed between 5.6.0 and 5.7.0?

Reply:
Sounds likely.

Reply:
I see that FA.json has a copy of the `Escape` method, which is different since the PR you mentioned. We could also align them, but that will effect other usage (for the good or the bad).